tests/clickhouse/optimize/test_optimize_scheduler.py

```diff
@@ -132,7 +132,7 @@ def test_subdivide_partitions(
     )
 
 
-last_midnight = (datetime.now() + timedelta(minutes=10)).replace(
+last_midnight = (datetime.now(UTC) + timedelta(minutes=10)).replace(
```

Comment:
this fixes a test that was broken separately from these changes (e.g. this fails if run in a non-UTC timezone on the master branch)